src/lib/migration-chunks/validation/editor-layout.ts

```diff
@@ -89,6 +88,53 @@ class AlreadyExistingFieldGroupCreates implements EditorLayoutValidation {
   }
 }
 
+class InvalidFieldGroupId implements EditorLayoutValidation {
+  validate (intent: Intent) {
+    if (!intent.isFieldGroupCreate() && !intent.isFieldGroupIdChange()) {
```

Comment:
I decided to not group the validations by intent this time to avoid duplicating the checks.